test/reducers/user_reducer_test.js

```diff
@@ -0,0 +1,41 @@
+import { expect } from 'chai'
+
+import userReducer from '../../web/static/js/reducers/user'
+
+
+describe('user reducer', () => {
+  describe('when no state is passed', () => {
+    it('should return the initial state', () => {
+      expect(userReducer(undefined, {})).to.deep.equal([])
```

Comment:
kinda hard to tell what the initial state is in this test. could you make it explicit via the `describe` or `it` descriptions?